testcontainers/tests/images.rs

```diff
@@ -314,6 +314,29 @@ fn generic_image_with_custom_entrypoint() {
     );
 }
 
+#[test]
+fn generic_image_exposed_ports() {
```

Comment:
What do you think of adding a test that fails because the port is _not_ exposed?

Reply:
Also a great idea, I've added a test.